virtcontainers/types/sandbox.go

```diff
@@ -52,6 +52,10 @@ type SandboxState struct {
 	// including the hypervisor are placed.
 	CgroupPath string `json:"cgroupPath,omitempty"`
 
+	// Path to all the cgroups setup for a container. Key is cgroup subsystem name
+	// with the value as the path.
+	CgroupPaths map[string]string `json:"cgroupPaths"`
```

Comment:
Perhaps I haven't spent enough time looking at the whole PR, but it isn't immediately clear to me the relationship between this map and the CgroupPath field already kept as part of the sandbox structure.

Can you clarify here, in the comments and/or in the commit?

Reply:
`CgroupPath` is used by `containerd/cgroup` and `CgroupPaths` by `libcontainer`.
`CgroupPaths` will replace to `CgroupPath` once we use `libcontainer` to create all the cgroups

Reply:
commit message updated, thanks

Reply:
nit: I know I asked for either or, but perhaps it'd be helpful to have it in the comment as well, so its easier to pick up on in future?